### DISCUSSION TITLE
Kiosk numeric input keypad for numeric post-usage questions

### DISCUSSION THREAD

**Discussion by r-xyz** [usnistgov/NEMO#129] [Q&A] [answered]
Hello,

Happy NEMO users here! :) I was reviewing the Feature Manual and I noticed the following numeric keypad for delayed logout:
![image](https://user-images.githubusercontent.com/100710244/208139326-de6c19f5-bd75-4168-9a2e-2786dc44695d.png)

I was wondering: would it be possible the same keypad to pop-up when inputting data on a numeric field? (I am thinking of kiosk post-usage questions).
Right now the only way to input data is using the really tiny arrow on the side of the box, but they are almost impossible to hit on the touchscreen we are using.

Thanks in advance for any feedback.

**Comment by rptmat57** [comment]
Hello,
It should be working on the kiosk already. Which version of NEMO are you on? and how is your post usage question setup?

**Reply by r-xyz** [reply]
Hi Mathieu,
Thanks for the quick reply.
It is version 4.3.0 (same behavior on 4.2.1).
The question is the last one of a set, like this:
```  
[{
...
},
{
      "type":"number",
      "max-width":200,
      "name":"Material Quantity",
      "consumable": "Consumable1",
      "suffix":"g",
      "placeholder":"0",
      "title":"If you used material from the common Stock, please enter the quantity."
   }
]
```

I tried different browsers (Chromium/Firefox), but the keypad do not seem to show up. 
Any suggestion on what else might be useful to debug it?

**Reply by rptmat57** [reply, marked as answer]
got it.
the name needs to be a unique name, valid for an input element and id, which means it should not have any spaces or `.` etc.
change it to `material_quantity` and it should work

**Reply by r-xyz** [reply]
Thanks, it works now!
My fault, I initially filled them thinking those would be displayed name and never changed again.
Now our kiosk is fully functional, thanks a lot for the help!
